.circleci/config.yml

```diff
@@ -1010,6 +1010,9 @@ workflows:
       - jinja2:
           requires:
             - flake8
+      - mako:
```

Comment:
Sorry, we want these to be listed alphabetically, can you move to be after `kombu` but before `molten` ?

Reply:
sorry, I did not notice that.